spec/index.html

```diff
@@ -577,6 +582,41 @@ <h3>RDF Version Announcement</h3>
   <p class="issue" data-number="70">
     Change "Classic Conformance" to "Basic Conformance" and define them as profiles.</p>
 
+  <section id="defined-version-labels">
+    <h3>Version Labels</h3>
+    <p>
+      A <dfn>version label</dfn> is a string that identifies the syntax and semantics conformance
+      for the RDF data.
+    </p>
+    <table id="tab-version-labels" class="simple">
+      <caption>Version Labels</caption>
+      <thead>
+        <tr>
+          <th style="text-align: center">Version Label</th>
+          <th style="text-align: center">Syntax</th>
+          <th style="text-align: center">Semantics</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>"1.2"</td>
+          <td>RDF 1.2 syntax</td>
+          <td><a href="https://www.w3.org/TR/rdf12-semantics/">RDF 1.2 Semantics</a></td>
+        </tr>
+        <tr>
+          <td>"1.2-basic"</td>
+          <td>RDF 1.2 syntax without triple terms</td>
```

Comment:
Should we add a description (note?) of what parsers are (not) expected to do when they encounter triple terms when the `1.2-basic` version is encountered?

In alignment to the optional nature of version announcement, I would expect that parsers are not required to error if they encounter triple terms under `1.2-basic`, but they may emit a warning.

Reply:
We need a separate discussion about that before we can add something in this section.

@rubensworks - Are you OK with merging this PR now and continuing the wider discussion? (I can raise an issue for it.)

The 1.10 ("RDF Version Announcement") has text about [advised behavior (paragraphs 2 and 3)](https://www.w3.org/TR/rdf12-concepts/#section-version-announcement). Paragraph 3 has a whole document assumption.

Personally, I think we need to be careful with guidance here. It can discourage new uses we haven't thought of.
There isn't deployment practice to record unlike the big note in the IRIs section.

Reply:
Ok, sounds good 👍